packages/boot/test/bootstrapTests/orchestration.test.ts

```diff
@@ -331,7 +331,15 @@ test.serial('basic-flows', async t => {
       '@agoric/builders/scripts/orchestration/init-basic-flows.js',
       [
         '--chainInfo',
-        JSON.stringify(withChainCapabilities(fetchedChainInfo)),
+        JSON.stringify(
+          withChainCapabilities({
```

Comment:
this is repeated below. It would help legibility and maintainability if this value were named like `fetchedChainInfo` is. please extract to const like `minimalSubsetChainInfo` or what have you.

I see it's used in multiple modules so consider making a new one for named chain info subsets which tests and contracts could import from.

Reply:
Just put this in `tools` for reuse